packages/typescript-estree/src/create-program/createWatchProgram.ts

```diff
@@ -548,4 +548,4 @@ function maybeInvalidateProgram(
   return null;
 }
 
-export { clearWatchCaches, createWatchProgram, getProgramsForProjects };
+export { clearWatchCaches, getProgramsForProjects };
```

Comment:
Not strictly necessary for this PR, but something I noticed - `createWatchProgram` isn't used anywhere else.

Reply:
#6088